helm/vitess/templates/_helpers.tpl

```diff
@@ -114,7 +125,7 @@ fi
 export EXTRA_MY_CNF="$FLAVOR_MYCNF:/vtdataroot/tabletdata/report-host.cnf:/vt/config/mycnf/rbr.cnf"
 
 {{ if . }}
-for filename in /vt/userconfig/*; do
+for filename in /vt/userconfig/*.cnf; do
```

Comment:
I'm curious how you're using `percona vault keychain configuration` here. Sounds interesting.

Reply:
I'm fine with this change as is, I just wasn't familiar with how the vault integration worked.